cpp/ql/src/semmle/code/cpp/internal/QualifiedName.qll

```diff
@@ -26,6 +26,22 @@ class Namespace extends @namespace {
     else result = this.getName()
   }
 
+  string getQualifierForMembers() {
```

Comment:
I was a bit confused at first until I figured out that this predicate can return multiple results by design. The existing comment sort of alludes to that, but something more explicit would be nice.